_script/test

```diff
@@ -6,4 +6,4 @@ cd $(dirname $0)/..
 
 echo "[*] Running Tests"
 
-go test -v -cover ./...
+go test -v -cover -race ./...
```

Comment:
👏  Nice! I've read https://blog.golang.org/race-detector and it's worthy to have it

Reply:
:+1: